service/project/FileState.kt

```diff
@@ -65,9 +66,8 @@ class FileState internal constructor(
     projectSrv: ProjectService,
 ) : PathState(parent, path, Type.FILE, projectSrv), Pageable.Runnable {
 
-    @OptIn(kotlin.time.ExperimentalTime::class)
     companion object {
-        private val LIVE_UPDATE_REFRESH_RATE: Duration = Duration.seconds(1)
+        private val LIVE_UPDATE_REFRESH_RATE: Duration = 1.seconds
```

Comment:
Unacceptable grammar. The Queen would not have been amused.
```suggestion
        private val LIVE_UPDATE_REFRESH_RATE: Duration = 1.second
```

Reply:
<img width="801" alt="image" src="https://user-images.githubusercontent.com/51956016/208319528-eec15063-ffaa-4e7c-9736-674d22c094d3.png">

![image](https://user-images.githubusercontent.com/51956016/208319694-b902bef1-1c2c-43b5-9d24-49e4b5e3b3ee.png)